main/guides/zoe/assets/trade-offer-safety-3.mmd

```diff
@@ -26,4 +26,4 @@ sequenceDiagram
 
     Zoe--)Zoe: escrow Price pmt
 
-    Zoe--)-C: joinHandler(gameSeat)
+    Zoe--)-C: tradeHandler(proceeds)
```

Comment:
looks like `proceeds` got mixed up with `buyerSeat`.

```suggestion
    Zoe--)-C: tradeHandler(buyerSeat)
```

Reply:
`gameSeat` is being referred to as `proceeds` in the current version of contract (the seat created via `makeEmptySeatKit`). Or am I interpreting it wrong? Or was it a typo in the previous iteration of the diagram?

Reply:
@dckc PTAL at this^

Reply:
> was it a typo in the previous iteration of the diagram?

yes, I suppose it was.

Reply:
Cool. Then I'll fix this for trade-offer-safety-4 as well. Thanks!